### PR TITLE
docs: mention empty string in `userDataDir`

### PR DESCRIPTION
**Pull request by szmarczak**
https://github.com/microsoft/playwright/blob/c4eb2d4b1e17a0de1cea7220be45545c7c524614/src/server/browserType.ts#L146-L149

### CODE REVIEW

File: docs/src/api/class-browsertype.md [hunk 1]
```diff
@@ -244,7 +244,8 @@ this context will automatically close the browser.
 Path to a User Data Directory, which stores browser session data like cookies and local storage. More details for
 [Chromium](https://chromium.googlesource.com/chromium/src/+/master/docs/user_data_dir.md#introduction) and
 [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#User_Profile).
-Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`.
+Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Use an empty string to
+generate and destroy it automatically.
```
Comment (szmarczak) [inline docs/src/api/class-browsertype.md:248]:
```suggestion
set a temporary directory instead.
```

Comment (szmarczak) [inline docs/src/api/class-browsertype.md:248]:
```suggestion
Note that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Pass an empty string to
use a temporary directory instead.
```

File: types/types.d.ts [hunk 2]
```diff
@@ -9656,7 +9656,8 @@ export interface BrowserType<Unused = {}> {
    * context will automatically close the browser.
    * @param userDataDir Path to a User Data Directory, which stores browser session data like cookies and local storage. More details for [Chromium](https://chromium.googlesource.com/chromium/src/+/master/docs/user_data_dir.md#introduction) and
    * [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#User_Profile). Note that Chromium's user
-   * data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`.
+   * data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Use an empty string to
+   * generate and destroy it automatically.
```
Comment (szmarczak) [inline types/types.d.ts:9660]:
```suggestion
   * data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. Pass an empty string to
   * use a temporary directory instead.
```

**Review by JoelEinbinder — APPROVED** [review]

**Review by dgozman — APPROVED** [review]
Thank you for the PR!

**Pull request merged**